st2common/st2common/rbac/types.py

```diff
@@ -91,6 +94,7 @@ class PermissionType(Enum):
     EXECUTION_RE_RUN = 'execution_rerun'
     EXECUTION_STOP = 'execution_stop'
     EXECUTION_ALL = 'execution_all'
+    EXECUTION_VIEWS_FILTERS_LIST = 'execution_view_filters_list'
```

Comment:
Naming - pack health would imply something about pack health while pack index health implies something about pack index health (which is more correct).